### PR TITLE
fix: React `ghostText` prop warning

### PR DESCRIPTION
**Pull request by hirehamir**

### CODE REVIEW

**Comment by hirehamir** [comment]
# Without this change

There's a warning in the terminal that runs `npm run dev`.

image

```
Warning: React does not recognize the `ghostText` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `ghosttext` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    at input
    at Input (webpack-internal:///./src/components/Input.tsx:8:18)
    at div
    at AuthShowcase (webpack-internal:///./src/pages/index.tsx:188:95)
    at div
    at div
    at main
    at Home (webpack-internal:///./src/pages/index.tsx:27:77)
    at SessionProvider (/home/hamir/repositories/party/node_modules/next-auth/react/index.js:454:24)
    at MyApp (webpack-internal:///./src/pages/_app.tsx:19:18)
    at Hydrate (file:///home/hamir/repositories/party/node_modules/@tanstack/react-query/build/lib/Hydrate.mjs:23:3)
    at QueryClientProvider (file:///home/hamir/repositories/party/node_modules/@tanstack/react-query/build/lib/QueryClientProvider.mjs:41:3)
    at TRPCProvider (file:///home/hamir/repositories/party/node_modules/@trpc/react-query/dist/createHooksInternal-d15d0073.mjs:193:17)
    at WithTRPC (webpack-internal:///./node_modules/@trpc/next/dist/index.mjs:45:83)
    at StyleRegistry (/home/hamir/repositories/party/node_modules/styled-jsx/dist/index/index.js:449:36)
    at PathnameContextProviderAdapter (/home/hamir/repositories/party/node_modules/next/dist/shared/lib/router/adapters.js:78:11)
    at AppContainer (/home/hamir/repositories/party/node_modules/next/dist/server/render.js:337:29)
    at AppContainerWithIsomorphicFiberStructure (/home/hamir/repositories/party/node_modules/next/dist/server/render.js:373:57)
    at div
    at Body (/home/hamir/repositories/party/node_modules/next/dist/server/render.js:673:21)
```

There's also no placeholder text. I assume the intent was for `ghostText` to be the placeholder text. Please correct me if I'm wrong.

image


# With this change

There isn't a warning for `ghostText` in the console.

image

File: src/pages/index.tsx [hunk 1]
```diff
@@ -79,10 +79,18 @@ const AuthShowcase: React.FC = () => {
       >
         {sessionData ? "Sign out" : "Sign in"}
       </button>
-      <Input type="text" id="id1" ghostText="This is our text field!" 
-      cssString="flex cursor-pointer items-center justify-between rounded bg-brand-gray_light px-3 py-1.5 text-brand-red hover:bg-brand-gray focus:outline-none" />
-      <Input type="textarea" id="id2" ghostText="This is our text area!" 
-      cssString="flex cursor-pointer items-center justify-between rounded bg-brand-gray_light px-3 py-1.5 text-brand-red hover:bg-brand-gray focus:outline-none" />
+      <Input
+        cssString="flex cursor-pointer items-center justify-between rounded bg-brand-gray_light px-3 py-1.5 text-brand-red hover:bg-brand-gray focus:outline-none"
```
Comment (elwerro) [inline src/pages/index.tsx]: `cssString` is supposed to be `className` too

File: src/pages/index.tsx [hunk 2]
```diff
@@ -79,10 +79,18 @@ const AuthShowcase: React.FC = () => {
...
+        id="id1"
+        placeholder="This is our text field!"
+        type="text"
+      />
+      <Input
+        cssString="flex cursor-pointer items-center justify-between rounded bg-brand-gray_light px-3 py-1.5 text-brand-red hover:bg-brand-gray focus:outline-none"
```
Comment (elwerro) [inline src/pages/index.tsx]: `className`

**Review by elwerro — APPROVED** [review]

File: src/pages/index.tsx (hunk 1 above)
Comment (hirehamir) [reply src/pages/index.tsx]: Thanks for catching that! The `<Input /`s look much nicer, now.

image

**Pull request merged**